### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by shivesh01**
Hey there, I have corrected grammar.
I hope you will like it.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Comment by ieahleen** [comment]
Thank you for opening this pull request.

This is a standard message notifying you that we've reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

Thank you and happy coding.

File: README.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ Once you have earned the Responsive Web Design, Algorithms and Data Structures,
 
 #### Legacy Certifications
 
-We also have 4 legacy certifications dating back to our 2015 curriculum, which are still available. All of the required projects for these legacy certifications will remain available on freeCodeCamp.org.
+We also have 4 legacy certifications dating back to our 2015 curriculum, which is still available. All of the required projects for these legacy certifications will remain available on freeCodeCamp.org.
```
Comment (nhcarrigan) [inline README.md:140]: Here, I believe `are` refers to the "4 legacy certifications", which is the correct usage.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 [freeCodeCamp.org](https://www.freecodecamp.org) is a friendly community where you can learn to code for free. It is run by a [donor-supported 501(c)(3) nonprofit](https://donate.freecodecamp.org) to help millions of busy adults transition into tech. Our community has already helped more than 10,000 people get their first developer job.
 
-Our full-stack web development and machine learning curriculum is completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
+Our full-stack web development and machine learning curriculum are completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
```
Comment (RandellDawson) [inline README.md:12]: Please revert this change as the `is` refers to the curriculum.  The "full-stack web development and machines learning" refer to the kind of curriculum we offer.
```suggestion
Our full-stack web development and machine learning curriculum is completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
```

Comment (nhcarrigan) [inline README.md:12]: This change makes "full-stack web development" and "machine learning" read as two separate curricula. IMHO, while the curriculum can be completed in independent sections, the idea is to create a unified learning resource. I'm on the fence with this verbiage change.,

File: README.md [hunk 3]
```diff
@@ -171,11 +171,11 @@ We appreciate any responsible disclosure of vulnerabilities that might impact th
 
 ### Contributing
 
-The freeCodeCamp.org community is possible thanks to thousands of kind volunteers like you. We welcome any and all contributions to the community and are excited to welcome you aboard.
+The freeCodeCamp.org community is possible thanks to thousands of kind volunteers like you. We welcome any contributions to the community and are excited to welcome you aboard.
```
Comment (RandellDawson) [inline README.md:174]: The following is another possibility, but I will leave the final decision to @QuincyLarson.
```suggestion
The freeCodeCamp.org community is possible thanks to thousands of kind volunteers like you. We welcome all contributions to the community and are excited to welcome you aboard.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
I have made two suggestions, but ultimately @QuincyLarson is responsible for approving all changes to our `README.md`.  He may choose to keep the existing document unchanged or choose one or both of these changes.

File: README.md (hunk 2 above)
Comment (RandellDawson) [reply README.md]: This particular sentence has been discussed before and last time it was decided `is` is the correct word.

**Review by raisedadead — APPROVED** [review]
